src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -11,7 +11,10 @@
     configure_logging_to_terminal,
 )
 from vivarium_census_prl_synth_pop.tools.jobmon import run_make_results_workflow
-from vivarium_census_prl_synth_pop.tools.make_results import build_results
+from vivarium_census_prl_synth_pop.tools.make_results import (
```

Comment:
Note: Since both `build_results` and `subset_results_by_state` are defined in the tools `__init__.py` above, you can import them directly like
```
from vivarium_census_prl_synth_pop.tools import (
    build_results,
    subset_results_by_state,
)
```